docs/api-irma-server.md

```diff
@@ -10,6 +10,7 @@ The API that this server offers consists of two parts:
 
 * [Endpoints under `/session`](#api-reference-requestor-endpoints) with which IRMA session requestors can start IRMA sessions, monitor their status and retrieve their result afterwards.
 * [Endpoints under `/irma`](#api-reference-irma-endpoints) for [frontend libraries](irma-frontend.md) and the [irmaclient](https://github.com/privacybydesign/irmago/tree/master/irmaclient)/[IRMA app](irma-app.md).
+* [Endpoints under `/irma/frontend`](#api-reference-irma-frontend-endpoints) exclusively for [frontend libraries](irma-frontend.md).
```

Comment:
Here and elsewhere we refer with `/irma/frontend` to these endpoints, while actually no such endpoints exist; instead they are of the form `/irma/session/{token}/frontend`. Is that not confusing?

Reply:
True, I didn't release that. I know refer to them as `Frontend endpoints` and I removed the `/frontend` prefix.